src/rebar3_ex_doc.erl

```diff
@@ -39,7 +39,7 @@ init(State) ->
                 {app, $a, "app", string, help(app)},
                 {ex_doc, $e, "ex_doc", string, help(ex_doc)},
                 {canonical, $n, "canonical", string, help(canonical)},
-                {output, $o, "output", {string, ?DEFAULT_DOC_DIR}, help(output)},
+                {output, $o, "output", {atom, undefined}, help(output)},
```

Comment:
Don't like this as it effects the help output, but having it set to doc by default makes the decision logic for output tricky.

Reply:
I resolved this via replacing the value in opts with `undefined` if `output == doc`. A little odd, but I think it's fine.